apps/root/src/services/transactionService.ts

```diff
@@ -149,27 +153,30 @@ export default class TransactionService extends EventsManager<TransactionService
   }
 
   addTransactionToHistory(tx: TransactionApiEvent) {
+    const transactionsHistory = structuredClone(this.transactionsHistory);
```

Comment:
why this and not a spread operator?

Reply:
I was trying a new approach with deep copying, but it wasn't necessary and gave me more trouble anyway. Now it's a spread operator :)